js/event_target.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import * as domTypes from "./dom_types";
+
+export class EventTarget implements domTypes.EventTarget {
+  public listeners: {
+    [type in string]: domTypes.EventListenerOrEventListenerObject[]
+  } = {};
+
+  public addEventListener(
+    type: string,
+    listener: domTypes.EventListenerOrEventListenerObject | null,
+    options?: boolean | domTypes.AddEventListenerOptions
```

Comment:
If we aren't implementing `options` we should not have it listed here, or at least comment that this is a `TODO`.

Reply:
yup I left a `TODO` at the top letting folks know this is an incomplete implementation. I'll check in with @ztplz otherwise I'll produce a follow-up full implementation